test/sequential/test-net-reconnect-error.js

```diff
@@ -27,22 +27,19 @@ const N = 20;
 let client_error_count = 0;
 let disconnect_count = 0;
 
-// Hopefully nothing is running on common.PORT
 const c = net.createConnection(common.PORT);
 
 c.on('connect', common.mustNotCall('client should not have connected'));
 
-c.on('error', function(e) {
-  console.error(`CLIENT error: ${e.code}`);
+c.on('error', common.mustCall((e) => {
   client_error_count++;
   assert.strictEqual('ECONNREFUSED', e.code);
-});
+}, 21));
 
-c.on('close', function() {
-  console.log('CLIENT disconnect');
+c.on('close', common.mustCall(() => {
   if (disconnect_count++ < N)
     c.connect(common.PORT); // reconnect
-});
+}, 21));
 
 process.on('exit', function() {
```

Comment:
[suggestion]
I see here an interesting point. Think about this handler...
On the one hand you have the `mustCall` count that verified each operation is called 21 time.
On the other hand you already have these numbers calculated, so why not check them.

From my perspective it's totally up to you to decide, but thinking about it IMHO is the important part.